.github/actions/torus-builder/entrypoint.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/sh
 
+set -e
```

Comment:
The `set -e` here was the fix for the build silently failing. Now, if any step in entrypoint.sh fails, the entire script will fail. Before, only the last command determined the exit status.

Also added -x so you can see what commands are being executed, makes debugging the build a bit easier.